_includes/modern-appendix.md

```diff
@@ -843,6 +851,32 @@ The text used in the last param of this message varies wildly.
 
 Indicates that there was a problem with a mode parameter. Replaces various implementation-specific mode-specific numerics.
 
+{% numericheader RPL_HELPSTART %}
+
+    "<client> <subject> :<firt line of help section>"
+
+Indicates the start of a reply to a {% command HELP %} command to the client.
+The text used in the last param of this message may vary, and SHOULD be displayed as-is by IRC clients to their users; possibly emphasized as a the title of the help section.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
+
+{% numericheader RPL_HELPTXT %}
+
+    "<client> <subject> :<line of help text>"
+
+When sending a reply to a {% command HELP %} command to the client, servers reply with each line of the help section as this numeric. Help lines MAY be wrapped to 80 characters by the server.
```

Comment:
Simplified text and changed the wrapping slightly. I think ours wraps at like 90 characters or something, makes sense to let servers choose how wide to wrap there.

```suggestion
Returns a line of {% command HELP %} text to the client. The text MUST be shown to the user as-is, and lines MAY be wrapped to a certain line length by the server.
```

Reply:
That was copy-pasted from RPL_MOTD's description

Reply:
Why did you make this a MUST? I don't think it's appropriate here, it doesn't break the protocol to transform it.

Reply:
Ah, hm. I'll toss in a PR updating `RPL_MOTD`'s description to match this, how's that sound?

Reply:
ok

Reply:
I think clients just show the help text as is, and especially since the help text can describe server-specific differences or notes it doesn't make sense to really let clients transform it. I'd be fine having it as a SHOULD or a MUST though, which would you prefer?

Reply:
(oh wait, do you mean 'transform' as in change whitespace or like replace messages with translated versions or something?)

Reply:
or interpret color codes, convert it to HTML or whatever. It doesn't matter to the protocol. See https://datatracker.ietf.org/doc/html/rfc2119#section-6

Reply:
ah, alright that makes sense. I was expecting like cases where clients would be substituting the actual text for their own (not applicable) text instead. give me a sec

Reply:
Hmm then it would be an issue. For example in weechat, `/help` shows the internal help instead of sending `HELP` to the server

Reply:
updated!